src/lava/magma/core/callback_fx.py

```diff
@@ -53,3 +54,23 @@ def post_run_callback(self,
                           board: NxBoard = None,
                           var_id_to_var_model_map: dict = None):
         pass
+
+
+class IterableCallBack(NxSdkCallbackFx):
```

Comment:
Actually a good observation. https://docs.python-guide.org/writing/gotchas/

As suggested making it a ty.Optional[ty.List] = None is better